drivers/thermal/hisi_thermal.c

```diff
@@ -351,7 +351,7 @@ static int hisi_thermal_probe(struct platform_device *pdev)
 		ret = hisi_thermal_register_sensor(pdev, data,
 						   &data->sensors[i], i);
 		if (ret)
-			dev_err(&pdev->dev,
+			dev_dbg(&pdev->dev,
 				"failed to register thermal sensor: %d\n", ret);
```

Comment:
Have fixed with new PR: https://github.com/96boards-hikey/linux/pull/17

On 16 June 2016 at 16:24, Guodong Xu <notifications@github.com> wrote:

> In drivers/thermal/hisi_thermal.c
> <https://github.com/96boards-hikey/linux/pull/16#discussion_r67305005>:
>
> > @@ -351,7 +351,7 @@ static int hisi_thermal_probe(struct platform_device *pdev)
> >  		ret = hisi_thermal_register_sensor(pdev, data,
> >  						   &data->sensors[i], i);
> >  		if (ret)
> > -			dev_err(&pdev->dev,
> > +			dev_dbg(&pdev->dev,
> >  				"failed to register thermal sensor: %d\n", ret);
>
> The two dev_dbg() are printing duplicate messages. Can you delete one?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/96boards-hikey/linux/pull/16/files/ad4fff1acec3de7b6a12ed56e8931714def424ac#r67305005>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AKsXkpBJAFpp2DriURyUcxR8Q0-z1pcNks5qMQhAgaJpZM4I3GLP>
> .
>